gravitee-apim-console-webui/src/management/settings/custom-user-fields/migrated/custom-user-fields-migrated.component.html

```diff
@@ -0,0 +1,115 @@
+      >
+        <mat-icon>add</mat-icon>
+        Add custom field
+      </button>
+    </mat-card-header>
+
+    <gio-table-wrapper
+      [disableSearchInput]="false"
+      [length]="nbTotalInstances"
+      [filters]="filters"
+      (filtersChange)="onFiltersChanged($event)"
+    >
+      <table mat-table matSort [dataSource]="customUserFieldsFiltered" aria-label="Custom user fields table">
+        <caption style="display: none">
+          Table with custom user fields.
+        </caption>
+
+        <ng-container matColumnDef="key">
+          <th mat-header-cell *matHeaderCellDef id="key" mat-sort-header>Key</th>
+          <td mat-cell *matCellDef="let element">
+            {{ element.key }}
+            @if (element.required) {
+              <span class="gio-badge-neutral">Required</span>
+            }
+          </td>
+        </ng-container>
+
+        <ng-container matColumnDef="label">
+          <th mat-header-cell *matHeaderCellDef id="label" mat-sort-header>Label</th>
+          <td mat-cell *matCellDef="let element">
+            {{ element.label }}
+          </td>
+        </ng-container>
+
+        <ng-container matColumnDef="values">
+          <th mat-header-cell *matHeaderCellDef id="values">Values</th>
+          <td mat-cell *matCellDef="let element">
+            @if (element.values?.length) {
+              @for (value of element.values; track value) {
+                <div>{{ value }}</div>
+              }
+            }
+          </td>
+        </ng-container>
+
+        <ng-container matColumnDef="actions">
+          <th mat-header-cell *matHeaderCellDef id="actions"></th>
+          <td mat-cell *matCellDef="let element">
+            <div class="actions">
+              <button
+                (click)="update(element)"
+                mat-button
+                aria-label="Edit custom field button"
+                matTooltip="Edit custom field"
+                [disabled]="isLoading"
+                *gioPermission="{ anyOf: ['organization-custom_user_fields-u'] }"
+              >
+                <mat-icon data-testid="custom-field_edit_icon">edit</mat-icon>
+              </button>
+
+              <button
+                (click)="delete(element)"
+                mat-button
+                aria-label="Delete custom field button"
+                matTooltip="Delete custom field"
+                data-testid="delete-button"
+                [disabled]="isLoading"
+                *gioPermission="{ anyOf: ['organization-custom_user_fields-d'] }"
+              >
+                <mat-icon data-testid="custom-field_delete_icon">delete</mat-icon>
+              </button>
+            </div>
+          </td>
+        </ng-container>
+
+        <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
+        <tr mat-row *matRowDef="let row; columns: displayedColumns"></tr>
+        <!-- Row shown when there is no data -->
+        <tr class="mat-mdc-row mdc-data-table__row" *matNoDataRow>
```

Comment:
I'd expect more than 2 possibilities here :
- loading.. - when isListLoading
- No data to display - when !isListLoading